docs/UserGuide.md

```diff
@@ -147,15 +147,35 @@ Examples:
 
 * `tag 1 t/SoftwareEng`
 
-### Searching tasks: `find`
+### Searching tasks with name: `find`
 
-Searches for a task with its associated tag.
+Searches for a task with a name provided.
 
 Format: `find KEYWORD`
 
 Examples:
 
-* `find CS2103T`
+* `find revise`
+
+### Searching tasks with tag: `findTag`
```

Comment:
Nice to see the new commands here. But I think the new mod command for finding by module should be included + the top list of commands and bottom summary should also be updated before the UG update is complete